app/application.js

```diff
@@ -1,18 +1,53 @@
 import Mn from 'backbone.marionette';
 import Backbone from 'backbone';
-import tracksView from 'views/tracks';
+import Tracks from './collections/tracks';
+import Playlists from './collections/playlists';
+import AppLayout from './views/app_layout';
 
+require('./styles/app.styl');
 
-class Application extends Mn.Application {
+const Application = Mn.Application.extend({
 
-    onStart () {
-        console.log('start');
+    initialize: function () {
```

Comment:
I suggest you stuck to [literal objects](http://babeljs.io/docs/learn-es2015/#enhanced-object-literals) syntax and sets the functions directly (i.e. `initialize () { ... }`). It'll be more readable and prevent the anonymized functions like this one in the stack trace.